models/workbench/src/index.ts

```diff
@@ -27,11 +27,14 @@ import workbench from './plugin'
 export { Application }
 
 @Model(workbench.class.Application, core.class.Doc, DOMAIN_MODEL)
-@UX(workbench.string.Application)
+@UX(workbench.string.Application, workbench.icon.Search)
 export class TApplication extends TDoc implements Application {
   label!: IntlString
   icon!: Asset
-  alias!: string
+
+  @Prop(TypeString(), core.string.Name)
```

Comment:
We need to order apps by some prop in qms table and display the name for that prop. But we could also duplicate it in QmsApp mixin to avoid this change

Reply:
Let's use _id natural order in model, no need for extra fields.

Reply:
Done